tests/neg/interleaving-ab.scala

```diff
@@ -0,0 +1,11 @@
+import scala.language.experimental.clauseInterleaving
+
+object Ab:
+    given String = ""
```

Comment:
```suggestion
object Ab:
  given String = ""
```
Same in many of the test files.

Reply:
Also: missing EOL at EOF, as reported by git and GitHub. Also same in many of the test files.